python-cffi/netplan/netdef.py

```diff
@@ -50,6 +50,15 @@ def dhcp4(self) -> bool:
     def dhcp6(self) -> bool:
         return bool(lib.netplan_netdef_get_dhcp6(self._ptr))
 
+    @property
+    def link_local(self) -> list:
+        linklocal = []
+        if bool(lib.netplan_netdef_get_link_local_ipv4(self._ptr)):
+            linklocal.append('ipv4')
+        if bool(lib.netplan_netdef_get_link_local_ipv6(self._ptr)):
+            linklocal.append('ipv6')
+        return linklocal
```

Comment:
praise: I link that design, which makes it very similar to the corresponding YAML stanza!